npm-shrinkwrap.json

```diff
@@ -3491,9 +3491,9 @@
           "resolved": "https://registry.npmjs.org/chalk/-/chalk-1.1.1.tgz",
           "dependencies": {
             "ansi-styles": {
-              "version": "2.2.0",
-              "from": "https://registry.npmjs.org/ansi-styles/-/ansi-styles-2.2.0.tgz",
-              "resolved": "https://registry.npmjs.org/ansi-styles/-/ansi-styles-2.2.0.tgz",
+              "version": "2.2.1",
```

Comment:
https://github.com/npm/npm/issues/17723

Reply:
Looks like we have CELA sign off

https://ossmsft.visualstudio.com/_oss?requestId=405768&_a=progress